index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Hola,cuál es su nombre?");
+alert(`Hola ${nombreUsuario}!`);
```

Comment:
(ejemplo) mejor si no usas interpolacion

```suggestion
alert("Hola" + nombreUsuario +"!");
```